### DISCUSSION TITLE
Implementation of Body Force in Peridynamics Module for Dynamic Fracture Modelling

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#19974] [Q&A General] [answered]
Dear MOOSE Community

I wish to simulate Dynamic Fracture for which constant stress needs to be applied on top and bottom of the domain but I am not sure how to implement that in MOOSE so any header in this regard will be beneficial.

Many Thanks
Pushkar

**Comment by GiudGiud** [comment, marked as answer]
@bwspenc on peridynamics

**Reply by GiudGiud** [reply]
@ppandit95 how's the status on this?

**Reply by ppandit95** [reply]
@GiudGiud I wanted to work on this but haven't got enough opportunity to look into it due to certain priorities and still I am not certain how this can be done by adding body force although I have looked into Body Force kernel but I guess it does not work with peridynamics module regarding which I may be wrong and in such a case it will be really helpful to get some leads in this regard.

Thanks
Pushkar

**Reply by GiudGiud** [reply]
Hello

So you want to apply load on the surfaces right? For this it wont be a body force kernel, most likely it will be a boundary condition. 
I would go through the tensor mechanics tutorial, I think we have something very similar there, possibly only one side for the application of the force. 

Guillaume
